dotCMS/src/main/java/com/dotmarketing/business/RoleAPI.java

```diff
@@ -247,6 +247,14 @@
 	 * @throws DotStateException
 	 */
 	public void addLayoutToRole(Layout layout, Role role) throws DotDataException, DotStateException;
+
+	/**
+	 * Returns true if the layout is already related to the role, otherwise false
+	 * @param layout {@link Layout}
+	 * @param role  {@link Role}
+	 * @return boolean
+	 */
+	public boolean hasLayoutToRole(Layout layout, Role role);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'hasLayoutToRole': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)